src/migration/1726147441577-SourceMembersCountFix.ts

```diff
@@ -0,0 +1,43 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class SourceMembersCountFix1726147441577 implements MigrationInterface {
+  name = 'SourceMembersCountFix1726147441577';
+
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    queryRunner.query(`
+      CREATE OR REPLACE FUNCTION increment_squad_members_count()
+        RETURNS TRIGGER
+        LANGUAGE PLPGSQL
+        AS
+      $$
+      BEGIN
+        UPDATE  source
+        SET     flags = jsonb_set(
+                          flags,
+                          '{totalMembers}',
+                          to_jsonb(COALESCE(CAST(flags->>'totalMembers' AS INTEGER), 0) + 1)
+                        )
+        WHERE   id = NEW."sourceId"
+        AND     type = 'squad';
+        RETURN NEW;
+      END;
+      $$
+    `);
+    queryRunner.query(
+      `
+      CREATE OR REPLACE TRIGGER increment_squad_members_count
+      AFTER INSERT ON "source_member"
+      FOR EACH ROW
+      WHEN (NEW.role != 'blocked')
```

Comment:
The only difference from the previous one is this condition.

https://github.com/dailydotdev/daily-api/blob/main/src/migration/1725981526888-SourceFlagMembersIncrement.ts

Reply:
down should have the old function and trigger definition :hidethepain: .. its so hard to update triggers in migrations

Reply:
Ah, right. Yep. Definitely needs to created on the down function 😅